### PR TITLE
Create generated indexes for all top level CLI submenus

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/2866

### CODE REVIEW

**Comment by jwitz** [comment]
keeping in draft while I change menu item descriptions

File: astro/cli/astro-completion.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 sidebar_label: "astro completion"
 title: "astro completion"
 id: astro-completion
-description: Reference documentation for astro completion.
+description: Generate completion scripts. 
 hide_table_of_contents: true
```
Comment (lzdanski) [inline astro/cli/astro-completion.md:6]: These all have the default page file/ folders for the autogenerated landing pages. We have a terminal icon that we should use instead to be consistent with [`astro dev` landing page.](https://deploy-preview-3043--gallant-neumann-56599d.netlify.app/astro/category/astro-dev) It'll probably be easiest for you to add it locally but here's a link to what to add in the front matter!
See - https://github.com/astronomer/docs/blob/58e8c47306c6a12c2b19ffca068b3b548acc2245/astro/cli/astro-dev-init.md?plain=1#L7
Screenshot 2023-11-02 at 11 42 56 AM
Screenshot 2023-11-02 at 11 43 47 AM

**Review by lzdanski — COMMENTED** [review]
Need to add the terminal icon!

File: astro/cli/astro-deployment-runtime-upgrade.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 sidebar_label: "astro deployment runtime upgrade"
 title: "astro deployment runtime upgrade"
 id: astro-deployment-runtime-upgrade
-description: Reference documentation for astro deployment runtime upgrade.
+description: Upgrade a Deployment (Software only)
```
Comment (lzdanski) [inline astro/cli/astro-deployment-runtime-upgrade.md:5]:
```suggestion
description: Upgrade a Deployment (Software only).
```

File: astro/cli/astro-deployment-service-account.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 sidebar_label: "astro deployment service-account"
 title: "astro deployment service-account"
 id: astro-deployment-service-account
-description: Reference documentation for astro deployment service-account
+description: Manage service accounts (Software only)
```
Comment (lzdanski) [inline astro/cli/astro-deployment-service-account.md:5]:
```suggestion
description: Manage service accounts (Software only).
```

File: astro/cli/astro-deployment-team.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 sidebar_label: "astro deployment team"
 title: "astro deployment team"
 id: astro-deployment-team
-description: Reference documentation for astro deployment team
+description: Manage Deployment Teams (Software only)
```
Comment (lzdanski) [inline astro/cli/astro-deployment-team.md:5]:
```suggestion
description: Manage Deployment Teams (Software only).
```

File: astro/cli/astro-deployment-user.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 sidebar_label: "astro deployment user"
 title: "astro deployment user"
 id: astro-deployment-user
-description: Reference documentation for astro deployment user.
+description: Manage Deployment users (Software only)
```
Comment (lzdanski) [inline astro/cli/astro-deployment-user.md:5]:
```suggestion
description: Manage Deployment users (Software only).
```

File: astro/cli/astro-workspace-service-account.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 sidebar_label: "astro workspace service-account"
 title: "astro workspace service-account"
 id: astro-workspace-service-account
-description: Reference documentation for astro workspace service-account.
+description: Manage Workspace service accounts (Software only)
```
Comment (lzdanski) [inline astro/cli/astro-workspace-service-account.md:5]:
```suggestion
description: Manage Workspace service accounts (Software only).
```

File: astro/view-logs.md [hunk 7]
```diff
@@ -114,97 +114,3 @@ To access task logs from the Cloud UI:
 4. Click a task run.
 5. Click **Instance Details**.
 6. Click **Log**.
-
-## Export task logs to Datadog
```
Comment (lzdanski) [inline astro/view-logs.md:118]: Not sure how this change got added to this PR - it might be for https://github.com/astronomer/docs/pull/3038?

**Review by lzdanski — COMMENTED** [review]
Some consistency things - feel free to disregard, but we have punctuation on all the descriptions except the ones with `(Software only)` in the desc. 

Also I think the view-logs file change snuck into this PR!

File: astro/cli/astro-deployment-worker-queue-create.md [hunk 8]
```diff
@@ -2,7 +2,7 @@
 sidebar_label: "astro deployment worker-queue create"
 title: "astro deployment worker-queue create"
 id: astro-deployment-worker-queue-create
-description: Reference documentation for astro deployment worker-queue create.
+description: Create a Deployment worker queue.
```
Comment (lzdanski) [inline astro/cli/astro-deployment-worker-queue-create.md:5]:
```suggestion
description: Create a Deployment worker queue.
sidebar_custom_props: { icon: 'img/term-icon.png' } 
```

File: astro/cli/astro-deployment-worker-queue-delete.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 sidebar_label: "astro deployment worker-queue delete"
 title: "astro deployment worker-queue delete"
 id: astro-deployment-worker-queue-delete
-description: Reference documentation for astro deployment worker-queue delete.
+description: Delete a Deployment worker queue.
```
Comment (lzdanski) [inline astro/cli/astro-deployment-worker-queue-delete.md:5]:
```suggestion
description: Delete a Deployment worker queue.
sidebar_custom_props: { icon: 'img/term-icon.png' } 
```

File: astro/cli/astro-deployment-worker-queue-update.md [hunk 10]
```diff
@@ -2,7 +2,7 @@
 sidebar_label: "astro deployment worker-queue update"
 title: "astro deployment worker-queue update"
 id: astro-deployment-worker-queue-update
-description: Reference documentation for astro deployment worker-queue update.
+description: Update a Deployment worker queue.
```
Comment (lzdanski) [inline astro/cli/astro-deployment-worker-queue-update.md:5]:
```suggestion
description: Update a Deployment worker queue.
sidebar_custom_props: { icon: 'img/term-icon.png' } 
```

File: astro/view-logs.md (hunk 7 above)
Comment (lzdanski) [reply astro/view-logs.md:118]: Thoughts about this change?

File: astro/cli/astro-workspace-service-account.md (hunk 6 above)
Comment (lzdanski) [reply astro/cli/astro-workspace-service-account.md:5]: I think this one might've gotten missed

**Review by lzdanski — COMMENTED** [review]
I think there might've been a couple things that got overlooked, but this is looking good!

File: astro/view-logs.md (hunk 7 above)
Comment (jwitz) [reply astro/view-logs.md:118]: Oh whoops I bet thats why it was missing from my other branch!

**Review by lzdanski — APPROVED** [review]

**Pull request merged**